### PR TITLE
Filter time

### PR DESCRIPTION
**Pull request by Adnan-961**
time
snip on iphone x 
@osamaakb please let me know if you need further changes

### CODE REVIEW

**Comment by osamaakb** [comment]
can you please close other PRs if you want to open a new one! Also, you don't have to create a new PR just fix the old one

File: src/components/FilterTime/FilterTime.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import React from "react";
+import { useState } from "react";
+import PropTypes from "prop-types";
+function FilterTime({ time }) {
+  let count = 0;
+  let notClicked =
+    "shadow-lg rounded-lg bg-gray-100 flex flex-col text-sm h-8 w-12 justify-center items-center border-solid border border-yellow-600 border-opacity-40 text-yellow-600";
+  let clicked =
+    "shadow-lg rounded-lg bg-yellow-600 flex flex-col text-sm h-8 w-12 justify-center items-center border-solid border border-yellow-700 text-white";
+  const [divClass, setClass] = useState(notClicked);
```
Comment (osamaakb) [inline src/components/FilterTime/FilterTime.jsx]: you can do a better naming for this like 
```suggestion
  const [isClickedStyle, setIsClickedStyle] = useState(notClicked);
```

**Review by osamaakb — CHANGES_REQUESTED** [review]
good job, just a small change

**Comment by Adnan-961** [comment]
@osamaakb done

**Review by osamaakb — APPROVED** [review]

**Pull request merged**